holochain-introduction-for-developers/holochain-brief-introduction.md

```diff
@@ -0,0 +1,79 @@
+# What is Holochain?
+
+> Holochain is a _protocol_ and _framework_ for creating cryptographically secure, portable, and distributed applications. It provides the necessary tools to allow individuals to maintain control over their identity and data, while interacting safely with a network of untrusted peers.
+>
+> The purpose of this document is to help developers and technologists quickly understand:
+>
+> * Why we're building Holochain
+> * What the project consists of
+> * What advantages it has over server-based and blockchain-based architectures
+
+## The problem
+
+The Internet is decentralized by design. Many services, such as the web and email, have no central points. But in order to move through the Internet confidently, we need **trust.** How can you know I am who I say I am, and how can you be sure that I'm acting with integrity?
+
+Centralized services act as proxies of trust. We may not be able to trust a million anonymous peers, but we can trust these big services to faithfully negotiate our interactions with those peers — **until we can't.** These systems are beginning to show weaknesses that are direct consequences of centralization.
+
+* **They are more attractive to attackers.**
+* **They leave users vulnerable to service outages.**
+* **They concentrate power in the hands of a few.**
+* **They burden the service operator with liabilities and costs.**
+
+Blockchain opened our imaginations to the possibility of trustable, decentralized coordination with untrusted peers, but suffers from efficiency problems that are inherent to its design. The next wave of decentralized ledgers improve performance and resilience but carry over some of blockchain's assumptions. They hint at the full potential of this technology but stop before they reach the finish line.
+
+## Holochain: empowered, distributed, trustable cooperation
```

Comment:
@zippy I changed a bunch of the language around sovereignty/mutuality, tried to remove the suggestion that this is for lone wolves only. Here I chose "empowered" and added "mutually" before "sovereign" in the following paragraph. Would love your feedback. Read on for more changes...